modules/custom/activity_basics/activity_basics.module

```diff
@@ -14,13 +15,32 @@ use Drupal\group\Entity\GroupContent;
  */
 function activity_basics_entity_insert(EntityInterface $entity) {
   _activity_basics_entity_action($entity, 'create_entitiy_action');
+
+  // Check if the entity created with "published" status.
+  if ($entity instanceof EntityPublishedInterface) {
+    if ($entity->isPublished()) {
+      _activity_basics_entity_action($entity, 'publish_entity_action');
+    }
+  }
 }
 
 /**
  * Implements hook_entity_update().
  */
 function activity_basics_entity_update(EntityInterface $entity) {
```

Comment:
I'd suggest adding a return type hint and deleting an error from the list of ignored by PHPStan.

Reply:
Fixed